spec/ic.did

```diff
@@ -43,6 +57,15 @@ service ic : {
   delete_canister : (record {canister_id : canister_id}) -> ();
   deposit_cycles : (record {canister_id : canister_id}) -> ();
   raw_rand : () -> (blob);
+  http_request : (record {
+    url : text;
+    method : variant { get };
+    headers: vec http_header;
+    body : opt blob;
+    transform : opt variant {
```

Comment:
Why is `transform` not declared as just `opt func`?

Reply:
In case we want to offer different ways of supplying a `transform` in the future.